index.d.ts

```diff
@@ -400,6 +400,58 @@ declare module 'plaid' {
     sort_code: string;
   }
 
+  interface StudentLoanStatus {
+    type: string | null;
+    end_date: string | null;
```

Comment:
Could we use `Iso8601DateString` for any of these new date fields?

Reply:
Good call — https://github.com/plaid/plaid-node/pull/212/commits/0e74db2853f75a7242a698c46a8e0ad5021a4209